doc/modules/preprocessing.rst

```diff
@@ -299,8 +302,39 @@ This can be confirmed on a independent testing set with similar remarks::
   ... # doctest: +ELLIPSIS +SKIP
   array([ 0.01...,  0.25...,  0.46...,  0.60... ,  0.94...])
 
-It is also possible to map the transformed data to a normal distribution by
-setting ``output_distribution='normal'``::
+Mapping to a Gaussian distribution
+----------------------------------
+
+In many modeling scenarios, normality of the features in a dataset is desirable.
+Power transforms are a family of parameteric transformations that aim to
+map data of any distribution to as close to a Gaussian distribution as possible.
+
+:class:`BoxCoxTransformer` provides one such transformation. The Box-Cox transform is given by:
```

Comment:
Please limit lines to under 80 chars where possible

Reply:
Fixed.